irdb/publish.py

```diff
@@ -64,21 +88,37 @@ def rename_package(pkg_path):
 
 
 def move_package(pkg_path, dir_name):
+    """
+    Move a package to a new location
+    """
     new_path = pth.join(dir_name, pth.basename(pkg_path))
     if pth.exists(new_path):
         os.remove(new_path)
     os.rename(pkg_path, new_path)
 
 
 def zip_package_folder(pkg_name):
-    pkg_dir = pth.join(PKGS_DIR, pkg_name)
-    # new_pkg_path = shutil.make_archive(pkg_name, "zip", pkg_dir, pkg_name)
-    new_pkg_path = shutil.make_archive(pkg_name, "zip", PKGS_DIR, pkg_name)
+    """
+    Create a zip file of packages in `pkg_names`
+
+    Directories `__pycache__` and hidden files (starting with `.`) are
+    ignored.
+    """
+    ignore_patterns = shutil.ignore_patterns("__pycache__", ".*")
```

Comment:
Should we also include docs here? Or should docs (e.g. ipynb files) also be compiled into the package?

Reply:
Btw, thanks for doing this. It's been on my to-do list for the last week, as I wasn't sure about whether we should include the docs folders in the compiled packages

Reply:
Notebooks should definitely be in the compiled package - on the website we will tell users how to download packages and where to find the notebooks there. OTOH readthedocs also renders the notebooks, right?